From 05c428550ef3d50f2b5663f78e130c4dfb44c57f Mon Sep 17 00:00:00 2001
From: Nathan Kinder <nkinder@redhat.com>
Date: Wed, 25 Feb 2015 14:22:02 -0800
Subject: [PATCH] Skip time sync during client install when using --no-ntp

When --no-ntp is specified during ipa-client-install, we still
attempt to perform a time sync before obtaining a TGT from the
KDC.  We should not be attempting to sync time with the KDC if
we are explicitly told to not configure ntp.

Ticket: https://fedorahosted.org/freeipa/ticket/4842
Reviewed-By: Martin Babinsky <mbabinsk@redhat.com>
---
 ipa-client/ipa-install/ipa-client-install | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipa-client/ipa-install/ipa-client-install b/ipa-client/ipa-install/ipa-client-install
index 70311ed6eb7df6a0143c9440403f7b479b6f39b0..1f0c565c33999527ffd7e34e4350235602014027 100755
--- a/ipa-client/ipa-install/ipa-client-install
+++ b/ipa-client/ipa-install/ipa-client-install
@@ -2075,7 +2075,7 @@ def install(options, env, fstore, statestore):
             root_logger.info("Removed old keys for realm %s from %s" % (
                 cli_realm, '/etc/krb5.keytab'))
 
-    if options.hostname and not options.on_master:
+    if options.hostname and not options.on_master and options.conf_ntp:
         # configure /etc/sysconfig/network to contain the hostname we set.
         # skip this step when run by ipa-server-install as it always configures
         # hostname if different from system hostname
-- 
2.4.3

